lib/bgs/service.rb

```diff
@@ -164,6 +164,16 @@ def user_ssn
       { ssn: @user.ssn }
     end
 
+    def get_regional_office_by_zip_code(zip_code, country, province, lob, ssn)
```

Comment:
can you include a test for this method in `spec/lib/bgs/service_spec.rb`?

Reply:
@erluti thank you for the suggestion - i've added a test for this as requested.